ci/cpu/build.sh

```diff
@@ -58,18 +59,20 @@ conda config --set ssl_verify False
 gpuci_logger "Build conda pkg for libcugraph"
 if [ "$BUILD_LIBCUGRAPH" == '1' ]; then
   if [[ -z "$PROJECT_FLASH" || "$PROJECT_FLASH" == "0" ]]; then
-    conda build conda/recipes/libcugraph
+    gpuci_conda_retry build --no-build-id --croot ${CONDA_BLD_DIR} conda/recipes/libcugraph
```

Comment:
i thought you had mentioned that `gpuci_conda_retry` didn't work with the `build` command. is that not the case?

Reply:
`gpuci_conda_retry` doesn't work when we use conda build to get the file output names for uploading, as it somehow has more output than intended. See cudf: https://github.com/rapidsai/cudf/blob/branch-0.19/ci/cpu/build.sh#L72